ipalib/messages.py

```diff
@@ -519,6 +519,18 @@ class ServerUpgradeRequired(PublicMessage):
     )
 
 
+class UidNumberOutOfLocalIDRange(PublicMessage):
+    """
+    **13034** UID Number is out of all local ID Ranges
+    """
+    errno = 13034
+    type = "warning"
+    format = _(
+        "User '%(user)s', with UID Number '%(uidnumber)d' is out of all ID "
+        "Ranges, 'ipantsecurityidentifier' will not be correctly generated."
```

Comment:
I would replace ipantsecurityidentifier with SID. Going into the implications of no SID is probably beyond this but I guess be prepared for the doc folks to ask.